workflow/wp-ba-results.csv

```diff
@@ -0,0 +1,16 @@
+gh_id, paper_doi, reuse_type, comment, citation_number, reused_doi, alt_url, page_num
```

Comment:
Why is this file here? Is this an incomplete result? is it done?

Reply:
I'm not entirely sure. I don't recall having moved/copied that file there.

Reply:
It's part of @andre-motta 's "-temp" commit 7d7f981f4f53632744cf3fbac95ae88800eba7ac

Reply:
This is the only point holding back this PR. Could you remove it for now and stash it until we clarified what this file was for?